feat(profiling): add wizard onboarding docs (#5778)
getsentry/sentry-docs: src/wizard/android/profiling-onboarding/android/2.configure-performance.md

---
name: Android
doc_link: https://docs.sentry.io/platforms/android/profiling/
support_level: production
type: language
---

#### Configure Performance

> Profiling depends on Sentry’s performance monitoring product being enabled beforehand. To enable performance monitoring in the SDK:

Reviewer: ```suggestion
Sentry’s performance monitoring product has to be enabled in order for Profiling to work. To enable performance monitoring in the SDK:
```
